### PR TITLE
added animal class and created a new animal in main

### PR DESCRIPTION
**Pull request by hvma411**

### CODE REVIEW

File: src/com/company/Animal.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class Animal {
+    String name;
```
Comment (PiumaP) [inline src/com/company/Animal.java:4]: add age

File: src/com/company/Main.java [hunk 2]
```diff
@@ -4,5 +4,9 @@
 
     public static void main(String[] args) {
         System.out.println("Hello it's a new repo");
+
+        Animal garfield = new Animal("Garfield", "Cat");
```
Comment (PiumaP) [inline src/com/company/Main.java:8]: create more animals

**Review by PiumaP — CHANGES_REQUESTED** [review]
make changes

File: src/com/company/Animal.java (hunk 1 above)
Comment (hvma411) [reply src/com/company/Animal.java:4]: done

File: src/com/company/Main.java (hunk 2 above)
Comment (hvma411) [reply src/com/company/Main.java]: done

**Pull request merged**